apostrophecms/a3-docs
# Pro 2939 piece page type doc

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR adds a new reference page documenting the piece-page-type. Closes PRO-2939.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly
1. Build the documentation
2. Navigate to reference->modules->piece-page-type.html

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
